docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst

```diff
@@ -26,16 +26,17 @@ Prerequisites
 ^^^^^^^^^^^^^
 
 To run the demo, you will need access
-to a Daml Enterprise release, the
+to a Daml Enterprise with drivers release, the
 :ref:`Canton docker repository <docker-instructions>`, as well as having docker, docker-compose, and Hyperledger Besu
 (`instructions here <https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Installation-Options/Install-Binaries>`_)
 installed.
 
+
 Introduction
 ^^^^^^^^^^^^
 
 The demo Ethereum deployment can be found inside the examples
-directory of the Canton Enterprise release. Unpack the Canton Enterprise release
+directory of the Canton Drivers release. Unpack the Canton Drivers release
```

Comment:
Is the "Canton Drivers" mentioned here the same release artifact that was called "Daml Enterprise with drivers release" few lines up? If yes, I would consider consolidating the nomenclature.

Reply:
amended